### PR TITLE
my side of the story

### PR DESCRIPTION
**Pull request by fredholmP**

### CODE REVIEW

**Comment by floridop** [comment]
Hi! Thanks for your submission.

Unfortunately it seems that there were changes in the main that conflict with yours. 
Could you please perform a merge as described in Tutorial 5 slides 82-96 and integrate those changes?
We do not want to lose any existing content, so you should keep what others have written and add yours.
You're free to reorganize the content blocks so that it still makes a story, but please do not modify
the content of what others have written.
You have time until the 13th of October at 14:40. 
If you can't make it by that time or you are done before the deadline, 
please add a comment in github so that I can progress with another student.

**Comment by fredholmP** [comment]
I have now made changes to make the stories compatible.

**Review by floridop — CHANGES_REQUESTED** [review]
Please check comment at line 37 and take action

**Comment by fredholmP** [comment]
I have merged the two stories as suggested.

File: fairytale.md [hunk 1]
```diff
@@ -32,8 +32,15 @@ he found himself in right now. He wondered how Xa-aX had managed to
 keep her cool in the face of what they had just witnessed at the 
 press room of the Johnson Space Center in Houston.
 
+
+
 There once lived a man, in Russia long ago, before this story was even
```
[the remark this answers is not in the document]
Comment (fredholmP) [reply fairytale.md]: I have merged the two stories as suggested.

[the remark this answers is not in the document]
Comment (fredholmP) [reply fairytale.md]: *Not merged as in a github merge but as in rewritten a combination of both of them in the more colloquial meaning.

**Comment by floridop** [comment]
Thanks for your contribution! Scenario A completed! See canvas for results

**Pull request merged**